src/api/tests/common.py

```diff
@@ -63,3 +64,16 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def create_user():
+    username = "restricted_user"
```

Comment:
popping in for a quick comment. All new functions need comments, so be sure to say how this is used etc if it's kept. Also I don't think we need this in general because the api is grabbing things for an individual user, so setting it once should be sufficient. Curious on the reasoning for adding this here. Also, don't user "restricted_user" or restricted in the email if this is supposed to a normal user. Restricted user is usually what we call someone who has an ineligible domain request

Reply:
ahh that makes sense! I will remove.  This was more of an experiment to see if the extra info was needed so some tests wouldn't fail.